test/config.test.js

```diff
@@ -103,6 +103,44 @@ class CaptureLogger {
   trace (message) { this._log('trace', message) }
 }
 
+function getEnvTable () {
+  const lines = fs.readFileSync(path.resolve(__dirname, '../lib/config.js')).toString().split('\n')
+  const matches = []
+  let state = 0
+  for (const line of lines) {
+    if (line.indexOf('ENV_TABLE = ') !== -1) {
+      state = 1
+      continue
+    }
+    if (state === 1 && line.trim() === '}') {
+      break
+    }
+    if (state === 1) {
+      matches.push(line)
+    }
+  }
+
+  if (matches.length === 0) {
+    return []
+  }
+  const vars = matches.map(function (item) {
+    const nameVars = item.trim().match(/^(.+?):.*?['"](.+?)['"]/)
+    if (!nameVars || nameVars.length !== 3) {
+      return null
+    }
+    return {
+      [nameVars[1]]: nameVars[2]
+    }
+  }).filter((item) => item)
+
+  const finalObject = {}
+  for (const pair of vars) {
+    finalObject[Object.keys(pair)[0]] = Object.values(pair)[0]
+  }
+  console.log(finalObject)
+  return finalObject
+}
+
```

Comment:
Could we export `ENV_TABLE` from "lib/config.js" for testing, instead of *parsing* "lib/config.js" here?

Reply:
Ha, that's funny.  I thought the exporting an additional symbol for this sort of linting might be considered spurious, which is why I did the parsing.  Changed this to use an exported and flattened `config.ENV_TABLE`.